Doc/faq/windows.rst

```diff
@@ -343,3 +343,6 @@ Simply rename the downloaded file to have the .TGZ extension, and WinZip will be
 able to handle it.  (If your copy of WinZip doesn't, get a newer one from
 https://www.winzip.com.)
 
+If you do not like or have WinZip or if you do not want to buy a copy of it
```

Comment:
Alright changed it to say for them to use their ``favorite file compression manager``